### ISSUE TITLE
Inclusion in Ansible 2.10.x

### ISSUE THREAD

**Issue by felixfontein** [CiscoDevNet/ansible-nso#4]
I was looking at this collcetion a bit. I have some questions/points:

1. example in https://github.com/CiscoDevNet/ansible-nso/blob/master/plugins/modules/nso_verify.py does not use FQCN
2. what's https://github.com/CiscoDevNet/ansible-nso/tree/master/tests/sanity/unused_ignores ?
3. do you have a CI running at least the sanity tests with ansible-base's stable-2.10 and devel branches?

Besides that, LGTM.

CC @gundalow @abadger

**Comment by gundalow** [comment]
If you create a PR with this lines https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml#L1-L116 in a new file called `.github/workflows/ansible-test.yml` then you will get Sanity and Unit tests running.

Any issues please should out and we can help.

**Comment by jabelk** [comment]
The examples are aligned to the NSO DevNet reservable sandbox instance and don't require a full FQCN

**Comment by felixfontein** [comment]
@jabelk examples are targeted at end-users, who will not be able to use the modules from the collections when they don't explicitly use FQCNs. I would assume that people will also use this collection outside your reservable sandbox.

Also, this is a mandatory requirement for inclusion in Ansible, see `All module and plugin EXAMPLES MUST:` in: https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#documentation

**Comment by jabelk** [comment]
Ah, I misunderstood. You mean the module call needs the full collection.modulename, not that the network device needs some type of renaming.

**Comment by jabelk** [comment]
I added `  cisco.nso.` namespace to the module example, thanks!

**Comment by mamullen13316** [comment]
@felixfontein prior to seeing your last comment on PR #142 I had submitted a new PR https://github.com/ansible/ansible/pull/72753 which I thought was possibly necessary for the migration based on further analysis of the Onyx module migration.

**Comment by felixfontein** [comment]
@mamullen13316 that's the very last thing that can be merged, and only after (a) this collection is included in Ansible 2.10, and (b) PR 142 in c.g is merged.

**Comment by mamullen13316** [comment]
@jabelk going through the [checklist](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst) it seems that we need to convert this repo to using branch name of main instead of master.  Would there be impact to others working on this repo,  DevNet labs,  or anything else that depends on the branch named master being present?

**Comment by felixfontein** [comment]
@mamullen13316 I don't think that's a requirement for collection repos outside the github.com/ansible-collections namespace. It is nice if collections outside of that namespace also do it, but not required to be part of Ansible.

**Comment by mamullen13316** [comment]
@felixfontein thanks for the clarification. In that case I believe we have satisfied the requirements in the checklist.

**Comment by felixfontein** [comment]
@mamullen13316 I think #5 needs to be addressed first.

**Comment by felixfontein** [comment]
Ok, I think everything's fine now. The only thing that is a bit strange is the https://github.com/CiscoDevNet/ansible-nso/tree/master/tests/sanity/unused_ignores directory, I think it can be safely deleted.

Maybe do another release (1.0.2) so that all fixes are included in the latest release? Then I think this collection can then be added to Ansible 2.10.

@gundalow what do you think?

**Comment by gundalow** [comment]
Yup, could you please delete that directory and release the next version?

Thanks for all the work that's been done here, and being responsive to the discussion.

**Comment by mamullen13316** [comment]
Deleted the unused_ignores and updated to 1.0.2 release.  You're welcome and thank you both for your help!

**Comment by mamullen13316** [comment]
@felixfontein should I do a PR on ansible-build-data to have cisco.nso included in https://github.com/ansible-community/ansible-build-data/blob/main/2.10/ansible.in?

**Comment by mamullen13316** [comment]
@felixfontein thanks for your help so far getting the nso modules migrated from community.network to their new home at cisco.nso.  Could you provide any guidance as to what we need to do in order to point the documentation on docs.ansible.com to the new repo?  It appears the docs are still pointing at community.network.  For example: https://docs.ansible.com/ansible/latest/collections/community/network/nso_config_module.html#ansible-collections-community-network-nso-config-module

**Comment by felixfontein** [comment]
@mamullen13316 the docs are for the latest 2.10.x release, and for 2.10 the modules are staying in community.network. The redirect will only happen for community.network 2.0.0, which will be included in Ansible 3.0.0 (to be released end of January / beginning of February). Ansible 2.10.5 (to be released today) will include the cisco.nso module, so users can manually use the modules from that collection already - and once the docsite is updated (latest tomorrow), the docs for cisco.nso will be in https://docs.ansible.com/ansible/latest/collections/cisco/nso/
